BrickBrackBruck/WalterDerHalter.cs

```diff
@@ -1,23 +1,121 @@
 using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+
 
 namespace BrickBrackBruck
 {
+
     class WalterDerHalter
     {
-        // List<
+        List<Ding> dinger = new List<Ding>();        
+
+        DateTime tag0 = new DateTime(1990,01,01);
+
+
+        public bool VerzeichnisHinzufügen(string pfad)
+        {
+            try
+            {
+                string[] Files = System.IO.Directory.GetFiles(pfad);
+
+                foreach (string f in Files)
+                {
+                    if (!DingHinzufügen( f ))
+                        return false;
+                }  
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("Fehler (VerzeichnisHinzufügen): " + ex.Message);   
```

Comment:
Gewohnheit von der Arbeit. Einmal bissel mehr Code reingeklatsch, dafür hat man in der Zukunft bei jedem Fehler direkt die Datei, die geladen wird. In der Regel gebe ich auch die Zeilenanzahl in der Exception mit aus, wenn eine Datei verarbeitet wird. Kommt dort so häufig vor, dass ich den Mehraufwand immer betreibe.